app/components/footer/footer.js

```diff
@@ -1,5 +1,8 @@
+const { shell } = require('electron')
```

Comment:
we're using ES6 import syntax, which made `npm test` fail
```suggestion
import { shell } from 'electron'
```